examples/local/README.md

```diff
@@ -0,0 +1,117 @@
+# Running cert-operator Locally
+
+**Note:** This should only be used for testing and development. See the
+[/kubernetes/][kubernetes-dir] directory and [Secrets][secrets-doc] for
+a production ready configuration.
+
+[kubernetes-dir]: https://github.com/giantswarm/cert-operator/tree/master/kubernetes
+[secrests-doc]: https://github.com/giantswarm/cert-operator#secrets
+
+This guide explains how to get running cert-operator locally. For example on
+minikube. Certificates created here are meant to be used by [aws-operator].
+
+All commands are assumed to be run from `examples/local` directory.
+
+[aws-operator]: https://github.com/giantswarm/aws-operator
+
+
+## Preparing Templates
+
+All yaml files in this directory are templates. Before proceeding this guide
+all placeholders must be replaced with sensible values.
+
+- *CLUSTER_NAME* - Cluster name to be created by [aws-operator].
+- *COMMON_DOMAIN* - Cluster name to be created by [aws-operator].
```

Comment:
@rossf7 I think I get something wrong with common domain. When we use it internally we use 2 domains. giantswarm.io for api & etcd, gigantic.io for ingress.

I think one of the cert files should be changed. I think it's also a reason why ingress LB isn't working on my local aws-operator setup. Is it worker-cert.yaml? Sould I add altName there?